README.md

```diff
@@ -214,6 +217,35 @@ const Title = styled('h3')`
 `;
 ```
 
+### Referring to other Components
+
+One limitation to fully encapsulated styles is that it's hard to contextually style components
+without them referencing each other. In astroturf you can use a component in a
+selector as if it were referencing a class selector.
+
+> Note: at the moment this only works for local identifiers, not imported components
```

Comment:
any timeframe on lifting this limitation?

Reply:
not at the moment no, It's a fairly hard problem to solve, resolving interdependent files statically. Right now tooling is very file based so we need to track imports and exports manually across babel compilations. Then there is just the problem that you may not be able to do that statically given how dynamic the language is, it's not always obvious how an export maps to a styled component.

Likely we will insist on some sort of naming convention that makes this a bit easier to infer without a ton of error prone analysis.